content/pages/exec.md

```diff
@@ -7,33 +7,39 @@ These are the people making things happen!
 
 You can view [past terms' committees here]({filename}/pages/past-exec.md).
 
-# Spring 2016 Committee #
+# Fall 2016 Committee #
 
 ## Chair ##
 
-Arshia Mufti (`a2mufti`)
+Jenny Sun (`j89sun`)
 
 ## Secretary ##
 
-Evy Kassirer (`eawkassi`)
+Anna Lorimer (`aemhlori`)
 
 ## Director of Finance ##
 
-Zarina Mamoukhouva (`zrmamouk`)
+Talia McCormick (`tdmccorm`)
 
 ## Big CSters Director ##
```

Comment:
I would rename this to "Directors". Someone should probably submit a second PR to fix the Winter 2016 exec post as well.